recipes/voro/build.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+mkdir -p ${PREFIX}/bin
+mkdir -p ${PREFIX}/lib
+
+make
+make install
```

Comment:
Probably not but I have to ask. Is there a `make check` that we can run here?

Reply:
thanks for the review - no , [there isn't yet](https://github.com/chr1shr/voro/blob/master/Makefile#L8) but I did ask for creating something like a test suite here
https://github.com/chr1shr/voro/issues/8#issuecomment-826649043

It should be straightforward but the developer currently seems busy